ui/app/AppLayouts/Wallet/popups/swap/SwapOutputData.qml

```diff
@@ -12,6 +12,8 @@ QtObject {
     // TODO: this should be string but backend gas_estimate_item.nim passes this as float
     property real totalFees: 0
     property bool hasError
```

Comment:
maybe we dont see this property anymore, we can use errCode instead to check if there was an eror.

Reply:
hmm kind of. we use `hasError` to indicate the response didn't contain any routes to show to the user, while `errCode` is the error code reported by the backend. We might have some routes reported with some error message, and have no routes returned but no error message, so I think it's fine to keep the two separate ones. I can change the property name to make the meaning more explicit though